File: public/usage-examples/graphics/draw_bitmap_named/draw_bitmap_named-1-simple.cpp
#include "splashkit.h"

int main()
{
    open_window("Basic Bitmap Drawing", 315, 330);

    load_bitmap("skbox", "skbox.png"); // Load bitmap image

    while (!quit_requested())
    {
        process_events();

        clear_screen(rgb_color(67, 80, 175));
        draw_bitmap("skbox", 50, 50); // draw bitmap image
        refresh_screen();
    }
    close_all_windows();

    return 0;
}

File: public/usage-examples/graphics/draw_bitmap_named/draw_bitmap_named-1-simple.py
from splashkit import *

# Open Window
open_window("Basic Bitmap Drawing", 315, 330)

# Load bitmap image
load_bitmap("skbox", "skbox.png")

while (not quit_requested()):
    process_events()

    clear_screen(rgb_color(67, 80, 175))
    draw_bitmap_named("skbox", 50, 50); # draw bitmap image
    refresh_screen()
close_all_windows()